tests/auth_tests/test_views.py

```diff
@@ -174,6 +174,15 @@ def test_email_found(self):
         # default functionality is 100% the same
         self.assertFalse(mail.outbox[0].message().is_multipart())
 
+    def test_extra_email_context(self):
+        """
+        Testing that the extra_email_context works
```

Comment:
Instead of "Testing that..." try to state the expected behavior: "extra_email_context should be available in the email template context."